app/api/helpers/utilities.py

```diff
@@ -1,7 +1,27 @@
 # PLEASE PUT ALL FUNCTIONS WHICH PERFORM GENERAL FORMATTING ON ANY DATATYPE WITHOUT USING ANY
 # MODULES RELATED TO THE EVENT-SYSTEM i.e FUNCTIONS RELATED TO DB MODELS, OTHER API MODULES
 # OR OTHER HELPER MODULES
+from dateutil import parser
+from pytz import timezone
 
 
 def dasherize(text):
     return text.replace('_', '-')
+
+
+def changeToTimezone(result):
+    tz = timezone(result['data']['attributes']['timezone'])
```

Comment:
@poush 
I tried it and it got it working fine without it.
For this looping , I used `'meta' ` key cause when there are many attributes it was there , but it didn't go well. So, I stick with it and it was working fine.
Are there any errors ?
( cause I took the idea from `call_for_papers` implementation and modified it somewhat , cause originally I tried it with `after_get_object` and realised that it couldn't be done as after the object is fetched and created you can't alter but only create a new one )